pkg/flare/archive_security_test.go

```diff
@@ -15,9 +15,8 @@ import (
 	flarehelpers "github.com/DataDog/datadog-agent/comp/core/flare/helpers"
 	"github.com/DataDog/datadog-agent/pkg/config"
 
-	// Required for the expvar init hooks to be called
-	// TODO: (components) Remove this when dogstatsd/server.go is migrated to a component
```

Comment:
I think this comment is still relevant.

Reply:
Ive updated the comment (below) as I don't think we can remove this as is. The required expvar is still initialized as a global (within the dogstatsd component package). 

I feel like this is going to be a minor challenge in other parts of the agent such that expvar's are "magically" referenced by name in many parts of the agent, which creates implicit dependency issues (like this one). 

Do we have a better long term plan for dealing with global expvars? I'd be happy to change the dogstasd server component accordingly to improve this.